<!DOCTYPE HTML>

<head>
		<title>Ramandeep Kaur</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<meta name="viewport" content="width=device-width, initial-scale=1">
		
		<style>
		.container {
		position: relative;
		width: 100%;
		overflow: hidden;
		padding-top: 56.25%; /* 16:9 Aspect Ratio */
		}

		.responsive-iframe {
		position: absolute;
		top: 0;
		left: 0;
		bottom: 0;
		right: 0;
		width: 100%;
		height: 100%;
		border: none;
		}
		.small-image {
			width: 80%;
			height: auto;
		}
		.resized-image {
			width: 80%;
			height: auto;
		}


		</style>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="logo">
							<i class='fas fa-laptop-code'></i>
						</div>
						<!-- <div class = "Image">
							<img src = "images/JSS_headshot.jpg"
							width = 500 height= auto />
                        </div> -->

						<div class="content">
							<div class="inner">
								<h1>Ramandeep Kaur</h1>
								<p>  I do magic with Stats and Commandline to Understand Plants 
								
								</p>								
							</div>
						</div>
						<nav>
							<ul>
								<li><a href="#intro">Who am I ?</a></li>
								<li><a href="#research">Research</a>
								
								<li><a href="#others">Fun</a></li>
								<li><a href="#databases">Databases</a></li>
								<li> <a href = "#teaching"> Teaching</a></li>
								<li> <a href = "#CV"> CV</a></li>
								
								<!--<li><a href="#elements">Elements</a></li>-->
							</ul>
						</nav>
					</header>

				<!-- Main -->
					<div id="main">

						<!-- Intro -->
							<article id="intro">
								<h2 class="major">My Story</h2>
								<span class="image main"><img src="images/ramandeep.JPG" width = 400 height= auto alt="" /></span>
								<p> Hello there! I'm Ramandeep Kaur, and I'm thrilled you dropped by! Currently, I'm pursuing my MS at South Dakota State University, USA. My roots trace back to a quaint farm in Punjab, India, where my journey with plants began. As a child, amidst the lush fields, my curiosity about the natural world blossomed. <br><br>

									Curiosity was my constant companion; I remember being a young girl, fascinated by the vibrant hues of mustard fields and the intricate patterns of rice paddies. I would often find myself pondering the secrets held within each seed and leaf. My father, an incredible source of inspiration, nurtured this curiosity, guiding me through the wonders of our farm. His stories and wisdom led me to pursue a B.TECH. in Biotechnology at Punjab Agricultural University, Ludhiana. 
									
									<br><br>University life was a kaleidoscope of new experiences, forging lifelong friendships and igniting a passion for plant breeding and evolutionary studies. This passion steered me towards a undergraduate Research at Punjb Agricultural University, where I had the fantastic opportunity to delve into the nuances of Bioinformatics and Biotechnology and Hrticulture in Dr. Amandeep Mittal's Lab.
									
									<br><br>My research journey took an exciting turn when I stumbled upon the fascinating world of soyabean roots and grapewines. Getting amazing experience and chance to work with two amazing Scientist Dr Senthil Subramanian  and Dr Anne Fennell, working on two different crops and a number of Projects.
									
									<br><br>Looking ahead, my dream is to contribute to Deciphering the complexities of Nature. A curious child I was turned into curious Women in STEM I am driven to find solutions through my expertise in genetics, bioinformatics, and molecular biology. With the amazing amalgamation of Computer Science and Molecular Biology, I aim to unlock the secrets of crop traits, genes, and cellular mechanisms that could revolutionize our approach to sustainable farming.</p>
									
							</article>

						<!-- Research -->
							<article id="research">
								<h2 class="major">Research</h2>
								<p>
									My overall research revolves around using pipettes and commandline to make sense from the complex plant pathways <br>
									
								</p>
								<nav>
									<ul>
										<li><a href="#MS.projects">MS Projects</a></li>
										<li><a href="#BT.projects">B.Tech Projects</a></li>
										
										
										
									</ul>
								</nav>
						
							<!-- M.S Projects -->
								<section id="MS.projects">
									<h3 class="major">M.S Projects</h3>
									
									<!-- Project 1 -->
									<p>
										<b>Project 1: miRNA analysis for dormancy induction in Vitis riparia grapevines.</b><br>
										Abstract: Grapevine is a temperate fruit crop that may grow in a variety of regions worldwide. Temperate perennials employ dormancy as one strategy to deal with persistent environmental challenges. Dormancy has a substantial impact on grapevine flowering and vegetative growth in the spring and helps the vines withstand the chilly winter weather. For the bud break and typical spring development to take place the next season, vines must be subjected to adequate hours of cool temperature exposure during the dormant phase. This environmental phenomenon is utilized in greenhouse, by exposing the grapevines to short day photoperiod treatment, to induce dormancy and get deeper insights into it. Physiological and transcriptome studies in grapevine and other species have revealed that various molecular alterations are observed during dormancy induction. According to previous research conducted in lab, the gene expression of V. riparia witnesses a drastic change in response to short photoperiod (SD; 13 h) treatment between 14 and 42 days, which explains why grapevines undergo a considerable transition from the start of dormancy (14 days) till dormant state (42 days). Yet, just a few research have looked at how microRNAs (miRNAs) control bud dormancy induction. This study is performed on two vine siblings VRS-F2 110 and VRS-F2 040 at 14 (2 weeks) and 42 days (6 weeks) with a goal to analyze the quantity and expression profiles of miRNAs during dormancy and use the target genes of these miRNAs to characterize putative regulatory networks connected to dormancy. This study will integrate genes, transcription factors, and molecular processes together to provide us a far greater understanding of Vitis riparia dormancy.
									</p>
									
									<p>
										<b>Project 2: Gene Regulatory Networks associated with high- and low-capacity nitrogen-fixing rhizobia.</b><br>
										<b>Abstract:</b> This research proposal delves into the intricate realm of gene regulatory networks (GRNs) linked to high- and low-capacity nitrogen-fixing rhizobia in the context of soybean roots. GRNs are complex molecular systems that control gene expression, with a pivotal role played by transcription factors (TFs). Unraveling these networks is critical for understanding the genetic mechanisms governing soybean nodulation and nitrogen fixation.
									</p>
									<p>
										Leveraging comprehensive transcriptomic data from split-root assays, this study harnesses computational modeling and data clustering to identify GRNs that are unique to the processes of soybean nodulation and nitrogen fixation. Co-expressed genes are organized into biclusters, shedding light on shared regulatory pathways that are active under specific conditions.
									</p>
									<p>
										The research primarily focuses on the symbiotic relationship between soybeans and rhizobia, where nitrogen-fixing rhizobia establish themselves in soybean roots, obtaining nutrients while simultaneously providing fixed nitrogen to the plant. This symbiotic interaction is essential for soybean growth and holds significant agricultural importance.
									</p>
									<p>
										<b>Key methodologies encompass the following:</b>
										<ul>
											<li>Aggregating data from transcriptomes to compile a list of TFs specific to root nodulation and nitrogen fixation.</li>
											<li>Employing biclustering using Qubic software to identify co-expressed gene modules.</li>
											<li>Utilizing Lemon-Tree and Inferelator, two module-based GRN inference methods, to infer GRNs.</li>
											<li>Combining regulatory predictions to establish a consensus GRN.</li>
											<li>Investigating potential protein-protein interactions (PPIs) via the STRING database.</li>
											<li>Analyzing cis-regulatory motifs and functional enrichment to provide evidence of direct regulation.</li>
											<li>This study promises to unravel the intricate web of gene regulation governing soybean nodulation and nitrogen fixation, offering valuable insights into enhancing crop productivity and sustainability.</li>
										</ul>
									</p>
									
									<!-- Project 3 -->
									<p>
										<b>Project 3: RNAseq analysis of shoot tip during photoperiod induced growth cessation in V. riparia grapevines.</b><br>
										Abstract: Short photoperiods usually cause growth cessation, cold adaptation, and dormancy induction in grapevines and other woody perennial plants native to temperate continental climes. Photoperiod-induced early activation of these mechanisms helps grapevines survive the winters in chilly temperate zones. Investigating the molecular processes in the shoot, will give researchers a better understanding of the signaling pathway that leads to growth arrest and dormancy induction. Vitis riparia Michx. grapevines that had grown for 35 days under a long photoperiod (long day, LD, 15 h) were given either a continuing LD or a short photoperiod (short day, SD, 10 h) treatment to begin studying photoperiod signal transduction. Shoot tips (10-node shoot terminals) were collected from each treatment at 7 and 21 days of LD and SD with 3 replicates of each treatment for RNASeq analysis. Differentially expressed genes were characterized in V. riparia shoot tips during active growth or SD induced growth cessation to examine physiological alterations in response to differential photoperiod treatments. The RNASeq analysis uncovered the signal transduction involved in V. riparia grapevine growth cessation and dormancy induction. Both 7SD and 7LD showed common growth pathways suggesting some level of similarity between the two. In 21SD, pathways involved in quenching reactive oxygen species were more abundant unlike LD tissues. In 7SD tissue, pathways involved in responding to stress conditions were more abundant than 21SD. Surprisingly, 21SD tissue was more abundant in pathways involving cell cycle and cell replication unlike 7SD. 
									</p>
									
									<!-- Project 4 -->
									<p>
										<b>Project 4: Promoter Analysis of differentially expressed genes between Vector control and STTM160 (miRNA 160 silenced) soybean roots.</b><br>
										Abstract: Legumes form root nodules to fix nitrogen through symbiotic relationship with nitrogen-fixing soil bacteria. In soybean, the root nodules form and function through hormonal balance of Auxin and Cytokinin. Auxin response factors (ARFs) regulate the auxin balance in soybean via miRNA 160 and 167, which silences ARF 16-2 and ARF 8a, accordingly. ARF 16-2 and ARF 8a binds to Auxin response elements (AuxREs) to repress and activate auxin gene expression respectively. Therefore, a balance between ARF8a and ARF 16-2 plays an important role in soybean nodule development. A RNASeq analysis was performed comparing auxin-responsive gene expression between control soybean roots and STTM160 roots (miRNA 160 silenced) to find genes and networks acting downstream of miR160 and miR167 during auxin response. miR160 was silenced using a short tandem target mimic (STTM160). Roots expressing a control construct (2C) and those expressing STTM160 were treated with a mock solution or auxin to identify auxin induced genes. The major gap in understanding the auxin response machinery is the identification of AuxRE motifs (transcription factors) recognized by ARF 16-2 and ARF 8a that control auxin gene expression. This study aims to evaluate promoters of these differentially expressed genes to identify distinct auxin response elements (AuxREs). We performed high-throughput promoter analysis to find AuxRE promoter motifs (TGTCNN) in differentially expressed genes and their binding affinities with ARF 16 and ARF 8a.


									</p>
								</section>


								<!-- B.Tech Projects -->
				<section id="BT.projects">
					<h3 class="major">B.Tech Projects</h3>
					<p>
						My undergraduate research was on making SSR and SNP markers and a genome database for Psidium guajava.
						<h4>Projects</h4>
						<ol>
							<li>
								<b>Guava Genome Database (GuavaINGDB):</b> A comprehensive Genome Database for Psidium Sp. <br>
								The Guava Genome Database is a repository of genetic and genomic data for the guava plant (Psidium guajava). It contains information on the plant's genome sequence, gene annotations, transcriptomics data, and more. Researchers can use the database to study the genetic basis of important traits such as fruit quality, disease resistance, and stress tolerance.<br>
								
							</li>
						</ol>
					</p>
				</section>

				<!-- Talks and Presentations -->
			<section id="talks-and-presentations">
				<h3 class="major">Talks and Presentations</h3>
				<span class="image main"><img src="images/present1.jpg" alt="Talks and Presentations" /></span>
				<p>
					I have had the privilege of sharing my research and insights through various talks and presentations. Here are some of the highlights:
					<ul>
						<li>
							<b>Poster presentation at Research Day, SDSU:</b> December 6, 2022.
						</li>
						<li>
							<b>Small oral talk at Research Day, SDSU:</b> April 28, 2023.
						</li>
					</ul>
    </p>
</section>


								
								
								
										
  
    <!-- \item \href {https://www.rural21.com/english/news/detail/article/root-trait-helps-plants-grow-deeper-roots-in-dry-soils.html?no_cache=1} {"Root trait helps plants grow deeper roots in dry soils”} by Rural21.com -->
									</ol>
									</p>
								</section>		
						</article> 

						<article id="teaching">
							<h2 class="major">Teaching</h2>
							<p>
								My job as a teacher is to inspire students to learn, their motivation and desire help me find the best way to teach. Some of the topics that I love to discuss are plant physiology, evolution, genetics, statistics, and human psychology.
							</p>
							
							
						
							<h3 class="major">Courses</h3>
							<ol>
								<li>
									PS 763: Crop Physiology – Graduate Course Fall 2022-2023 <br>
									Recordings of the lectures are are attached. <br><br>
									<i>Crop physiology
										Course Description: Systems analysis of factors which limit or increase crop production and the
										potential for qualitative and quantitative adjustments.
										</i>
									<br><br>
									<!-- Replace image with YouTube video -->
							<div class="container"> 
								<iframe class="responsive-iframe" src="https://www.youtube.com/embed/DLHlcmzsMhk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> 
							</div>
								</li>
								<br>
								<li>
									STAT 435/535: Applied Bioinformatics  – Graduate Course Spring 2022-2023<br>
									Recordings of the lectures are are attached. <br><br>
									<i>Applied bioinformatics
										This course is an introduction to bioinformatics for students with biological backgrounds. Using real data as examples, a project-based strategy will be adopted throughout the class so that students can understand algorithms in the context of the biological problems. Students will be exposed to public databases, major algorithms, websites and software.
										</i>
									<br><br>
									<!-- Replace image with YouTube video -->
							<div class="container"> 
								<iframe class="responsive-iframe" src="https://www.youtube.com/embed/0H7h5uGvDQM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> 
							</div>
								</li>
								
							</ol>
						
							<!-- Add a Mentoring section -->
							<h3 class="major">Mentoring</h3>
							<p>Mentored a REU student (summer intern) 2022 to conduct bioinformatics’ analysis for her REU research project – “Central and peripheral zone-specific transcriptomes during soybean nodule initiation”.</p>
						
							
						</article>
						

								
						<!-- Others -->
							<article id="others">
								<h2 class="major">Fun</h2>
								<p>
									Executive Member and Media Head of International Dance Crew at South Dakota State University
								</p>
								<span class="image main"><img src="images/dance1.jpg" alt="" /></span>
								<span class="image main"><img src="images/dance2.jpg" alt="" /></span>
								<span class="image main"><img src="images/dance3.jpg" alt="" /></span>


								</article>
						
					
								<article id="databases">
								
									  <h2 class="major">Database Development</h2>
									  <div class="row">
										<div class="col-md-6">
										  <h3>GuavaINGDB : Guava Genome Database</h3>
										  <img src="images/guava.png" alt="Guava Genome" class="img-fluid mb-3 small-image">

										  <p>The Guava Genome Database is a repository of genetic and genomic data for the guava plant (Psidium guajava). It contains information on the plant's genome sequence, gene annotations, transcriptomics data, and more. Researchers can use the database to study the genetic basis of important traits such as fruit quality, disease resistance, and stress tolerance.</p>
										  <a href="www.guavaingdb.com" class="btn btn-primary">Go to Database</a>
										</div>
										<br>
										<div class="col-md-6">
										  <h3>CitrusMarks: Citrus Marker Database</h3>
										  <img src="images/citrus.png" alt="CitrusMarks" class="img-fluid mb-3 resized-image">

										  
										  <p>CitrusMarks is a database of DNA markers for citrus plants, including sweet orange, mandarin, and grapefruit. DNA markers are used to identify specific genetic variations that are associated with desirable traits, such as fruit flavor, color, and size. CitrusMarks contains information on thousands of markers, along with their physical positions on the genome, and can be used by breeders to develop new citrus varieties with improved characteristics.</p>
										  <a href="ww.citrusmarks.com" class="btn btn-primary">Go to Database</a>
										</div>
										<br>
										<div class="col-md-6">
											<h3>Indian Medicinal Genome Database (Under Development)</h3>

											<p>
												The Indian Medicinal Genome Database is an innovative and comprehensive resource dedicated to the genomic study of Indian medicinal plants. This database is a treasure trove of genetic information, focusing on plants traditionally used in Ayurveda, Siddha, Unani, and other indigenous medicinal systems. It features a vast collection of genomic sequences, gene annotations, and DNA markers linked to therapeutic traits like healing properties, biochemical composition, and resistance to pests and diseases.

												Aimed at revolutionizing the field of phytomedicine, the database serves as a critical tool for researchers, herbalists, and pharmaceutical companies. It facilitates the exploration of genetic diversity, aids in the conservation of rare medicinal species, and supports the breeding of plants with enhanced medicinal qualities. The Indian Medicinal Genome Database is not just a platform for scientific discovery but also a step towards preserving and validating the rich heritage of Indian traditional medicine.
											</p>

										  </div>
									  </div>
									  
									
								  </article>
					
								
								<h3 class="major">Contact </h3>
								<ul class="icons">
									<li><a href="mailto:jzs806@psu.edu" class="fa fa-envelope"><span class="label"></span></a></li>
									<li><a href="https://twitter.com/Jgdp_S_Sidhu?ref_src=twsrc%5Etfw" class="icon brands fa-twitter" target="_blank"><span class="label">Twitter</span></a></li>
										<!-- <li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
										<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li> -->
									<li><a href="https://github.com/jagdeep2021" class="icon brands fa-github" target="_blank"><span class="label">GitHub</span></a></li>
									<li><a href="https://www.linkedin.com/in/jagdeep-sidhu-718754138" class="icon brands fa-linkedin" target="_blank"><span class="label">Linkedin</span></a></li>
									<li><a href="https://plantscience.psu.edu/directory/jzs806" class="fa fa-globe" target="_blank"><span class="label"></span></a></li>
									
									
								</ul>
							</article>

						<!-- CV -->
						<article id="CV">
							<h2 class="major">CV</h2>
							
							<iframe src="images/RK_CV.pdf" #view=fitH width="100%" height="600px">
							</iframe>
						</article>

						<!-- Elements -->
							<article id="elements">
								<h2 class="major">Elements</h2>

								<section>
									<h3 class="major">Text</h3>
									<p>This is <b>bold</b> and this is <strong>strong</strong>. This is <i>italic</i> and this is <em>emphasized</em>.
									This is <sup>superscript</sup> text and this is <sub>subscript</sub> text.
									This is <u>underlined</u> and this is code: <code>for (;;) { ... }</code>. Finally, <a href="#">this is a link</a>.</p>
									<hr />
									<h2>Heading Level 2</h2>
									<h3>Heading Level 3</h3>
									<h4>Heading Level 4</h4>
									<h5>Heading Level 5</h5>
									<h6>Heading Level 6</h6>
									<hr />
									<h4>Blockquote</h4>
									<blockquote>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan faucibus. Vestibulum ante ipsum primis in faucibus lorem ipsum dolor sit amet nullam adipiscing eu felis.</blockquote>
									<h4>Preformatted</h4>
									<pre><code>i = 0;

while (!deck.isInOrder()) {
    print 'Iteration ' + i;
    deck.shuffle();
    i++;
}

print 'It took ' + i + ' iterations to sort the deck.';</code></pre>
								</section>

								

							</article>

					</div>

				<!-- Footer -->
					<footer id="footer">
						<p> Ramandeep Kaur, South Dakota State University, Brooking SD <br>
							<ul class="icons">
								<li><a href="https://twitter.com/ramandeep_03" target="_blank" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
								<!-- <li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li> -->
								<!-- <li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li> -->
								<li><a href="mailto:ramandeep.kaur@jacks.sdstate.edu" target="_blank" class="fa fa-envelope"><span class="label"></span></a></li>
								<li><a href="https://www.linkedin.com/in/ramandeep-kaur-290307/" class="icon brands fa-linkedin" target="_blank"><span class="label">Linkedin</span></a></li>
								<li><a href="https://www.sdstate.edu/directory/ramandeep-kaur" class="fa fa-globe" target="_blank"><span class="label"></span></a></li>
									
							</footer>

			</div>

		<!-- BG -->
			<div id="bg"></div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>


</html>
